src/helpers/requireFiles.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
 const resolve = require('resolve');
+const fromEntries = require('object.fromentries');
 
 module.exports = function requireFiles(arg) {
-  [].concat(arg).forEach((requirePath) => {
+  if (arg && !Array.isArray(arg) && typeof arg === 'object') {
+    return arg;
+  }
+  return fromEntries([].concat(arg).map(requirePath => [
```

Comment:
What's the purpose of concat here?

Reply:
If its an array, use it, else if it’s not an array, wrap it in one.

Reply:
i could also do `flatten([arg], 1)` i suppose